woocommerce-gateway-ebanx/woocommerce-gateway-ebanx.php

```diff
@@ -251,25 +251,43 @@ public function my_account_template()
 		{
 			if (isset($_POST['credit-card-delete']) && is_account_page()) {
```

Comment:
We're not using superglobals anymore, please replace `isset($_POST[<value>])` with `WC_EBANX_Request::has(<value>)`